app/views/products/show.html.erb

```diff
@@ -82,7 +82,24 @@
 
 </div><!-- /.row -->
 
+<% if @product.readme? %>
+<hr />
+<div class="container row">
+  <button class="btn" data-toggle="collapse" data-target=".readme">Toggle Readme</button>
```

Comment:
1. Instead of a toggle, just show the contents of the README. Add a header above with the title the name of the readme, using HTML like: <div class="page-header"><h3>README.md</h3></div>
2. Follow the bootstrap guidelines: a div that is a row should always include a div that is a col. I think that "container" class is superfluous here.
3. That col should have the offset to the left so that the contents of the README line up with the rest of the attributes  (see bottom of comment)
4. Check and verify apps.awesim.org, especially on the "Shared apps" where the Permissions controls appear underneath the metadata. The permissions controls should appear above the README. In fact the permission controls should probably have the same offset as the readme and the metadata.


```html
<div class="row">
    <div class="col-md-offset-2 col-md-10 col-sm-10 col-xs-12"> <!-- col-md-6 or col-md-8 or col-md-10 -->
      <div class="page-header"><h3>README.md</h3></div>
```

<img width="1087" alt="screen 2018-07-31 at 3 33 55 pm" src="https://user-images.githubusercontent.com/512333/43482652-4b75c8ca-94d7-11e8-9ac0-989c9fec956e.png">
<img width="1088" alt="screen 2018-07-31 at 3 33 41 pm" src="https://user-images.githubusercontent.com/512333/43482653-4b8d170a-94d7-11e8-8268-82b1e86ccc92.png">